contract/test/build-proposal.test.js

```diff
@@ -20,7 +22,10 @@ test.before(async t => {
    * @param {string} name of an _already cached_ bundle
    */
   const compressBundle = async name => {
-    const bundle = await bundleCache.load('', name);
+    // NOTE load options must match those used in the proposal builder
+    const bundle = await bundleCache.load('', name, trace, {
+      elideComments: true,
+    });
```

Comment:
Nicely done. Thanks for maintaining this prototype tool, even after applying `elideElements` uniformly throughout agoric-sdk.

Reply:
@dckc ,
`elideElements`?

Reply:
typo for `elideComments`